flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/aggregate/GroupAggFunction.java

```diff
@@ -83,61 +85,57 @@
 	// stores the accumulators
 	private transient ValueState<RowData> accState = null;
 
+	private final StateTtlConfig ttlConfig;
```

Comment:
nit: We can use `long stateRetentionTime` as the member variable to save the serialized state. I find you already declare stateRetentionTime in other classes, so would be better to keep consistent.

Reply:
get

Reply:
@wuchong sorry, I looked the code again, `ttlConfig` object also used in `processElement` method, so it must be global object. However, it only used in `open` method in other classes, to keep consistent, I think we can set ttlConfig as global object in other classes, what do you think？

Reply:
I think we can simply use `stateRetentionTime > 0` to check whether ttl is enabled. We can also wrap that as a utility if needed.